examples/catvdog/README.rst

```diff
@@ -0,0 +1,52 @@
+Cat and Dog Image Classifier
+============================
+
+Requirements
+------------
+
+* Follow the documented `installation instructions <https://klio.readthedocs.io/en/latest/quickstart/installation.html>`_.
+* Python 3.6+
+
+.. note::
+
+    Make sure the minor version matches with the base Docker image stated on the ``FROM`` line in the `Dockerfile <./Dockerfile#L2>`_.
+    Adjust the declared Python version in the base Docker image to match the minor version of Python you wish to use
+
+Run
+---
+
+.. code-block:: sh
+
+    # Clone example
+    $ git clone https://github.com/spotify/klio
+    $ cd klio/examples/catvdog
+
+    # Create a klio-job.yaml based off of the example config
+    $ export GCP_PROJECT=<your gcp project>
+    $ sed -e "s/\$GCP_PROJECT/${GCP_PROJECT}/" \
```

Comment:
just a thought: I wonder if we could incorporate this functionality into the `klio job config set` command, like adding functionality to permanently apply a template parameter.  Could be useful but maybe not worth the effort.

Reply:
ah hmm yeah I like that idea from a user perspective, but that could tightly couple some things (`pipeline_options.project` -> pubsub topics/subscriptions / gcs buckets / bigquery tables - and what if we're reading a topic/data outside the project?). Maybe it's possible to do it a bit more cleanly 🤔